### PR TITLE
Manuals in Spanish

### PR DESCRIPTION
**Pull request by MasQueElite**
Because I'm that cool.

### CODE REVIEW

**Comment by MasQueElite** [comment]
Test

**Comment by MasQueElite** [comment]
Just add the _Added new transl.__ (I think I messed it up)

**Comment by MasQueElite** [comment]
*it

File: HTML/Chess translated (Español — Ajedrez) (MásQuéÉlite).html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Mod de Keep Talking and Nobody Explodes</span>
+                <span class="page-header-section-title">Ajedrez</span>
+            </div>
+            <div class="page-content">
+                <img src="img/Component/Chess.svg" class="diagram">
+                <h2>En caso de ajedrez</h2>
+                <p class="flavour-text">Bajo presión, el ajedrez se puede sentir como un juego de Hundir la flota.</p>
+                <p>Este módulo se basa en un <b>tablero de ajedrez 6×6</b> <i>(referenciado en la siguiente página)</i> y todas las piezas siguen las normas estándar de movimiento de la FIDE.
```
Comment (shortc1rcuit) [inline HTML/Chess translated (Español — Ajedrez) (MásQuéÉlite).html:33]:
```suggestion
                <p>Este módulo se basa en un <strong>tablero de ajedrez 6×6</strong> <em>(referenciado en la siguiente página)</em> y todas las piezas siguen las normas estándar de movimiento de la FIDE.
```

Comment (MasQueElite) [reply HTML/Chess translated (Español — Ajedrez) (MásQuéÉlite).html:33]: Why it is more recommendable to use 'strong' and 'em' instead of 'b' and 'i'?

Comment (MasQueElite) [reply HTML/Chess translated (Español — Ajedrez) (MásQuéÉlite).html:33]: I'm in a message box lol

Comment (shortc1rcuit) [reply HTML/Chess translated (Español — Ajedrez) (MásQuéÉlite).html:33]: Not sure tbh but it's recommended in "Red's guide to updating the repo"
https://docs.google.com/document/d/1F1v94J1EpPJqtK4Km_7GJ1mJgH0keWdISoxvhWVB1tI/edit

Comment (MasQueElite) [reply HTML/Chess translated (Español — Ajedrez) (MásQuéÉlite).html:33]: Oh, I didn't find that document. Thank you!

**Pull request merged**